### ISSUE TITLE
Tabbed Interface - Carousel: 

### ISSUE THREAD

**Issue by shawnthompson** [wet-boew/wet-boew#6661]
News.gc.ca has implemented a second carousel on their [homepage](http://news.gc.ca/web/index-en.do#wb-auto-2), which is acting very weird in mobile (tested iPhone 4C, 5 and 6). The carousel keeps changing even when it's paused. It seems to reach to the touch event while you scroll. I have tried it using just the base of GCWeb and wet-boew builds and it gave me all the same results. I can not seem to figure out what is going on with it.

Has anyone ever came into this problem or could someone try to debug it for me?

**Comment by thomasgohard** [comment]
@shawnthompson that's an interesting bug. I've reproduced on both Android and iPhone.

On a side note, you need to optimise that page. The page is 2 MB (1.7 MB of which are images) and takes about 30 seconds to load. On a laptop. On the three phones I tested with it took between about 3 to 5 minutes to load.

Also, the panels in the first carousel don't all have the same height. This causes the content to jump around when the panels change.

**Comment by shawnthompson** [comment]
@thomasgohard I agree with you that the page is heavy but it's there any documentation stating that it needs to be under a certain size?

I was just reporting the problem. I don't actually make changes on the site. 

Cc @biljanap

**Comment by pjackson28** [comment]
@shawnthompson There is the 200kb target and 500kb limit specified in the Content & IA spec. That page size is 4 times the upper limit which will mean major problems for some users (particularly mobile users).

**Comment by pjackson28** [comment]
I've played around with it in Android and it looks like redrawns are being triggered as the rest of the page flickers when this happens (even outside of the carousels). I wonder if it has to do with the CSS or Bootstrap classes used inside the tab panel. Maybe how the content is implemented is causing the content to be wider than the tab panel which then is triggering horizontal scrolling touch when touched (due to the built in CSS transitions). There have been numerous issues previously where how the content was implemented caused horizontal clipping/scrolling issues. By far the most common cause was improper use of Bootstrap classes, such as using row without a container or some other combination that made the content wider than than the tab panel (due to negative padding and/or margins).

That could be the case here so check to make sure the content is not wider than the tab panel and that Bootstrap classes and CSS are used properly for the tabpanel.

**Comment by pjackson28** [comment]
Actually looking at this closer, I suspect it is a combination of the following:
* Nesting of wb-eqht (so equal heights is being applied on both an ancestor and dependent, on both the tabpanels element and inside each tabpanel.
* Likely improper use of Bootstrap classes. For instance there are col-* classes without a parent row element and grandparent container element.
* Custom CSS (e.g., "cn-news-container") that is forcing the same CSS across all viewport sizes (so not not surprising there are issues in certain viewport sizes).

So this looks very much like an implementation issue (improper Bootstrap usage, custom CSS, possible improper use of equal heights). In addition, there is the possibly of a compatibility issue with equal heights (whether there is an issue with it being in the tabbed interface or with nesting of equal heights).

**Comment by hsrudnicki** [comment]
@shawnthompson @thomasgohard @pjackson28 : Can this issue be closed?

**Comment by hsrudnicki** [comment]
Made "Going Stale" in order to close this issue asap as resolved/fixed (particularly if the other issue(s) about equal heights have been resolved...)  or if otherwise? Either way, please update this issue's status, thank you

**Comment by hsrudnicki** [comment]
Is this still an issue? Either way, please update this issue's status, whether this issue can be closed as resolved/fixed or if otherwise?  Thank you

**Comment by hsrudnicki** [comment]
Is this still an issue? Please update the status of this whether this can be closed or otherwise, thx

**Comment by hsrudnicki** [comment]
@shawnthompson and others:  Is this still an issue/a problem in this year?

**Comment by hsrudnicki** [comment]
Based on the above, can be this referenced with the later issues: #7522, and maybe #7591 ? Please keep updated the status of this, thank you
